src/Services/Core/MemberService.php

```diff
@@ -454,5 +454,89 @@ public function decryptTokenForReset($token)
         }
 
         return $payload->id;
-    }
-}
\ No newline at end of file
+	}
+
+	public function export($idList)
+	{
+		if (!isset($idList)) {
+			throw new KerosException("idList not specified.", 400);
+		}
+
+		$members = $this->getSome($idList);
+
+		if (empty($members)) {
+			throw new KerosException("No members found.", 404);
+		}
+
+		$filepath = "documents/tmp/members" . (new \DateTime("NOW"))->format("Y-m-d;H:i:s") . ".csv";
+		$csvFile = fopen($filepath, "w+");
+
+		$tags = array(
+			'Pole' => 'positions',
+			'Nom' => 'lastName',
+			'Prenom' => 'firstName',
+			'Sexe' => 'gender',
+			'Annee'=> 'schoolYear',
+			'Departement' => 'department',
+			'Telephone' => 'telephone',
+			'Adresse Mail' => 'email',
+			'Recrutement' => 'createdDate',
+			'Statut' => 'status',
+			'Naissance' => 'birthday'
+		);
+		fputcsv($csvFile, array_keys($tags));
+
+		foreach ($members as $member) {
+			$memberData = $member->jsonSerialize();
+			$data = array();
+			foreach ($tags as $value){
+				switch($value) {
+					case 'schoolYear':
+					case 'firstName':
+					case 'lastName':
+					case 'telephone':
+					case 'email':
+					case 'birthday':
+						$data[] = $memberData[$value];
+						break;
+					case 'createdDate':
+						$data[] = $memberData[$value]->format('Y-m-d');
+						break;
+					case 'gender':
+					case 'department':
+						$data[] = $memberData[$value]->getLabel();
+						break;
+					case 'positions':
+						$latestYear = $memberData[$value][0]->getYear();

```

Comment:
Si le membre a pas de position ça plante 
`[2020-02-27 13:18:07] Keros.ERROR: File : C:\Users\Paul\Documents\Keros-Back\src\Services\Core\MemberService.php. Line : 510. Message : Call to a member function getYear() on null. [] []`